packages/core/examples/multiVolumeCanvasToWorld/index.ts

```diff
@@ -105,7 +105,7 @@ async function run() {
     type: ViewportType.ORTHOGRAPHIC,
     element,
     defaultOptions: {
-      orientation: ORIENTATION.SAGITTAL,
+      orientation: ORIENTATION.AXIAL,
```

Comment:
This is intentional to show to be SAGITTAL as it shows it is a volume

Reply:
nice catch, I changed it for testing and forgot to put it back